README.md

```diff
@@ -195,3 +195,23 @@ In chrome browser:
 `localhost::<port>/<uri_prefix>/<file_name>`
 
 `<file_name>` is optional. It depends on the type of handler. 
+
+## Deployment
+
+`make build` to build and run webserver in docker
```

Comment:
Make build actually builds the large image file into a tar file using Dockerfile, then it saves that tar file as webserver.tar.